test/e2e/node/kubelet_authz.go

```diff
@@ -42,24 +43,24 @@ var _ = SIGDescribe(feature.KubeletFineGrainedAuthz, func() {
 
 	ginkgo.Context("when calling kubelet API", func() {
 		ginkgo.It("check /healthz enpoint is accessible via nodes/healthz RBAC", func(ctx context.Context) {
-			sc := runKubeletAuthzTest(ctx, f, "healthz", "healthz")
+			sc := runKubeletAuthzTest(ctx, f, "healthz", "healthz", true)
 			gomega.Expect(sc).To(gomega.Equal("200"))
 		})
 		ginkgo.It("check /healthz enpoint is accessible via nodes/proxy RBAC", func(ctx context.Context) {
-			sc := runKubeletAuthzTest(ctx, f, "healthz", "proxy")
+			sc := runKubeletAuthzTest(ctx, f, "healthz", "proxy", false)
 			gomega.Expect(sc).To(gomega.Equal("200"))
 		})
 		ginkgo.It("check /healthz enpoint is not accessible via nodes/configz RBAC", func(ctx context.Context) {
-			sc := runKubeletAuthzTest(ctx, f, "healthz", "configz")
+			sc := runKubeletAuthzTest(ctx, f, "healthz", "configz", true)
 			gomega.Expect(sc).To(gomega.Equal("403"))
 		})
 	})
 })
 
-func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint, authzSubresource string) string {
+func runKubeletAuthzTest(ctx context.Context, f *framework.Framework, endpoint, authzSubresource string, failOnAuthzUpdateErr bool) string {
 	ns := f.Namespace.Name
-	saName := authzSubresource
-	crName := authzSubresource
+	saName := authzSubresource + "-sa-" + rand.String(5)
+	crName := authzSubresource + "-cr-" + rand.String(5)
```

Comment:
optional: use `GenerateName` instead

Reply:
I remove the commit changing this in favor of https://github.com/kubernetes/kubernetes/pull/131249